requirements-test.txt

```diff
@@ -7,3 +7,5 @@ pytest-github-actions-annotate-failures==0.1.8
 pytest==7.2.2
 PyYaml==5.4.1
 responses==0.23.1
+setuptools==67.7.2
+wheel==0.40.0
```

Comment:
Added separately in 5b549c58d2309f0b3a4056e0b0a7d4bc874405ba to keep CI happy (unrelated failures on python 3.12). I'll refactor the build to declarative so we can move away from `setup.py` and we hopefully don't need these anymore.